recipe/meta.yaml

```diff
@@ -47,6 +48,7 @@ test:
   commands:
     - pip check
     - openfe --help
+    - openfe --version
```

Comment:
this checks the version via the cli, but I think they're defined in one place

Reply:
I am not sure what you mean, this is how we get the version: https://github.com/OpenFreeEnergy/openfe/blob/main/openfecli/__init__.py#L8

Which is the same code path here: https://github.com/OpenFreeEnergy/openfe/blob/main/openfe/__init__.py#L26